modules/AWS/eks/nodegroup_launch_template/locals.tf

```diff
@@ -4,34 +4,34 @@ locals {
   # If the region is found in any of the supported regions lists, it will be used as kinesis region in osquery flags.
   # Otherwise either the default european or non european region and role are used
 
-  aws_sts_arn_role_eu       = "${var.kinesis_log_producers_role_arns["eu"]}"
-  aws_sts_arn_role_non_eu   = "${var.kinesis_log_producers_role_arns["non-eu"]}"
+  aws_sts_arn_role_eu     = var.kinesis_log_producers_role_arns["eu"]
+  aws_sts_arn_role_non_eu = var.kinesis_log_producers_role_arns["non-eu"]
```

Comment:
Just for my own learning, why was this removed?

Reply:
fmt removed redundant ${} as it's typically used for interpolation which isn't the case here.